### ISSUE TITLE
Getting warnings related to loading the config "react-intl"

### ISSUE THREAD

**Issue by phillipj** [evenchange4/react-intl.macro#83]
Hi!

I'm doing a create-react-app upgrade from v1 -> v3 these days and we've been happy users of [react-intl-cra](https://github.com/evenchange4/react-intl-cra) -- thanks for that btw! 💯

While upgrading, react-intl-cra had a `babel-preset-react-app` version conflict, so I decided to try transitioning to the macro approach used by this project instead. After having updated all the places I found in our project that used to import `defineMessages | FormattedMessage | FormattedHtmlMessage` from `react-intl`, I get a little closer to successfully building a production bundle.

What I'm seeing now is a lot of these warnings printed:

```
$ MESSAGE_DIR='./.messages' npx react-scripts build

Creating an optimized production build...
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.

...
```

Have anyone seen these warnings before by any chance? There are no error thrown, so there's really much extra info to get, other than the related [babel-plugin-macros source code](https://github.com/kentcdodds/babel-plugin-macros/blob/2d57c60adadd8904db9624614faffc34a251c40d/src/index.js#L296) suggesting there `optionsConfig.options === undefined` is the culprit 🤷‍♂ I don't have any prior macros experience, so not really sure what to read from that.

**Comment by phillipj** [comment]
Forgot to mention, I do get .json files created in the `./messages` directory. Though whenever I see warnings like this, I do assume something is not working properly and has to be fixed.

**Comment by phillipj** [comment]
Ohmy gotta love weird npm graph issues. The above was fixed by explicitly installing an up-to-date version of the [browerslist](https://github.com/browserslist/browserslist) package:

```
$ npm i browserslist@latest -D
```

*found a hint about the above in this [Stackoverflow answer](https://stackoverflow.com/a/55872461)*.

I got a hint of doing that because of the log *before* the warnings, a line I removed from my initial issue description because I assumed it had nothing to do with this project -- in other words, be careful with assumptions 😄 

```
$ MESSAGE_DIR='./.messages' npx react-scripts build

Creating an optimized production build...
Browserslist: caniuse-lite is outdated. Please run next command `npm update`
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.
There was an error trying to load the config "react-intl" for the macro imported from "react-intl.macro. Please see the error thrown for more information.
```

Bottom line; closing this issue since it probably wasn't related afterall, sorry for bothering you!
